README.md

```diff
@@ -1,28 +1,67 @@
 # cached-module-loader
 
-Highly experimental bundler and loader of Node.js modules.
+Caching bundler and loader of Node.js modules which uses the `cachedData` feature in the `vm` module (available since Node.js v6).
 
-**Don't use this.**
+☣ **Highly experimental, don't use it.**
 
-## Usage ;-)
+## Installation
```

Comment:
I used `Installation` instead of `Install`, because there's a `Usage` down the road. Imo `Install` would look good if it was `Use` too.